bitnami/cassandra/templates/_helpers.tpl

```diff
@@ -280,3 +288,41 @@ WARNING: JVM New Heap Size not set in value jvm.newHeapSize. When not set, the c
      MAX(Memory Limit (if set) / 64, 256M)
 {{- end }}
 {{- end -}}
+
+{{/*
+Dynamic Seed Discovery Init-Container
+*/}}
+{{- define "cassandra.dynamicSeedDiscovery" -}}
+- name: dynamic-seed-discovery
+  image: {{ include "cassandra.dynamicSeedDiscovery.image" .}}
+  imagePullPolicy: {{ .Values.dynamicSeedDiscovery.image.pullPolicy | quote }}
+  securityContext:
+      runAsUser: 0  # Run as root user
+      privileged: true
+  command:
+      - "/bin/sh"
+      - "-c"
+      - |
+        export DYNAMIC_SEED_DIR=/opt/bitnami/cassandra/tmp &&
+        apt update -y && apt install -y netcat-traditional dnsutils &&
+        dig +short {{ .Release.Name }}-headless.{{ .Release.Namespace}}.svc.{{ .Values.clusterDomain }} | grep -v $POD_IP | \
+          while read NODE_IP; do \
+            nc -zvw3 $NODE_IP 9042 && \
```

Comment:
I understand `9042` is the CQL port, is that correct? If so, please use the value from `values.yaml`.